cmd/kubeadm/app/constants/constants.go

```diff
@@ -464,14 +464,14 @@ var (
 
 	// SupportedEtcdVersion lists officially supported etcd versions with corresponding Kubernetes releases
 	SupportedEtcdVersion = map[uint8]string{
-		22: "3.5.10-0",
-		23: "3.5.10-0",
-		24: "3.5.10-0",
-		25: "3.5.10-0",
-		26: "3.5.10-0",
-		27: "3.5.10-0",
-		28: "3.5.10-0",
-		29: "3.5.10-0",
+		22: "3.5.11-0",
+		23: "3.5.11-0",
+		24: "3.5.11-0",
+		25: "3.5.11-0",
+		26: "3.5.11-0",
+		27: "3.5.11-0",
+		28: "3.5.11-0",
+		29: "3.5.11-0",
```

Comment:
/cc @SataQiu
maybe we can handle the upgrade in this PR?

Reply:
Link #122265